msgpack/src/test/scala/fs2/data/msgpack/ValidationSpec.scala

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2024 fs2-data Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fs2
+package data
+package msgpack
+
+import cats.effect._
+import low.MsgpackItem
+import weaver._
+import scodec.bits._
+import cats.implicits._
+
+object ValidationSpec extends SimpleIOSuite {
+  def validation1[F[_]: Sync](cases: (MsgpackItem, Throwable)*): F[Expectations] =
+    Stream
+      .emits(cases)
+      .evalMap { case (lhs, rhs) =>
+        Stream
+          .emit(lhs)
+          .through(low.toBinary[F])
+          .compile
+          .drain
+          .redeem(expect.same(_, rhs), _ => failure(s"Expected error for item ${lhs}"))
+      }
+      .compile
+      .foldMonoid
```

Comment:
We can just delegate to `validation` here:
```suggestion
    validation[F](cases.map { case (i, t) => List(i) -> t }*)
```